src/repository.rs

```diff
@@ -248,11 +270,78 @@ impl<M: Model> Repository<M> {
     ///     .unwrap();
     /// assert_eq!(bulk_update_res.nb_affected, 2);
     /// assert_eq!(bulk_update_res.nb_modified, 2);
+    /// # }
+    /// # let mut rt = tokio::runtime::Runtime::new().unwrap();
+    /// # rt.block_on(demo());
```

Comment:
Since we only care about checking compilation, this could be removed as well
```suggestion
    /// # let mut rt = tokio::runtime::Runtime::new().unwrap();
    /// # rt.block_on(demo());
```